Functional adjustment and adding translation

jfkuayue opened the pull request:
1. Adjustment of cargo aging: reefer from 720 to 1200 (newer reefers have better cold effects), luxury sleeper to 800, ordinary sleeper to 500-600, limited express trains to 250-400, and increase the cargo_age_period of old passenger coaches. (well, it might be bad news for metagamers)
2. Implement of different reliability decay speeds for different engines.
3. Add Simplified Chinese translation.

jfkuayue committed:
Functional adjustment and adding translation

1. Adjustment of cargo aging: reefer from 720 to 1200 (newer reefers have better cold effects), luxury sleeper to 800, ordinary sleeper to 500-600, limited express trains to 250-400, and increase the cargo_age_period of old passenger coaches.
2. Implement of different reliability decay speeds for different engines.
3. Add Simplified Chinese translation.

WenSimEHRP commented:
The translations should be added later, as Jake wrote in README:
> 4.0 Developer Notices
JP+ Engines overwrites vehicle IDs from JapanSet3 Trains. This is subject to change in the future and will involve a major shakeup of IDs.\
\
The lang file contains many redundant strings as it is still being reworked. _Translations will not be accepted until the restructure is complete, to prevent wasted time_.

Thus, I think you should consider removing the trasnlations, although they are usable.

jfkuayue committed:
Deleting translation

1. Deleting Simplified Chinese translation;
2. Extending cargo_age_period for torpedo and slag ladle wagons, since they are preferred by factories despite slow speed (Timberwolf has done such bonus)

EmperorJake commented:
I think the lang file is just about done anyway, so I could remove that part of the readme and start accepting translations. Though it might still change a bit as I will keep adding more variants. If you could reopen the translation as a separate PR that would be great 🙂

JohnFranklin523 commented:
> I think the lang file is just about done anyway, so I could remove that part of the readme and start accepting translations. Though it might still change a bit as I will keep adding more variants. If you could reopen the translation as a separate PR that would be great 🙂

Okay, I have deleted translations in this PR.
